dom0/sd-dom0-files.sls

```diff
@@ -60,12 +79,13 @@ dom0-remove-securedrop-workstation-stretch-template:
       - file: dom0-workstation-rpm-repo
 
 dom0-install-securedrop-workstation-template:
```

Comment:
I think that this entire block will need to be in a conditional, since `pkg.installed` is the required syntax for 4.0, and `qvm.template_installted` is required for 4.1